src/js/views/patients/worklist/bulk-edit/bulk-edit_views.js

```diff
@@ -86,12 +86,10 @@ const BulkEditActionsHeaderView = View.extend({
       </div>
       <div>
         <button class="button--icon u-margin--r-8 js-menu">{{far "ellipsis-h"}}</button>
-        <button class="button--icon js-close">{{fas "times"}}</button>
```

Comment:
The sidebar is a modal and already has a close button. This one was covering up that button. Tests were looking for the modal close button and couldn't click it because it was covered.

Reply:
so the ellipsis menu fits right up there next to the modal cleanly?  no vertical alignment issues?